frontend/src/metabase/ui/components/buttons/Button/Button.styled.tsx

```diff
@@ -56,12 +56,12 @@ export const getButtonOverrides = (): MantineThemeOverride["components"] => ({
 
         return {
           root: {
-            color: theme.fn.themeColor("text-dark"),
-            borderColor: theme.fn.themeColor("border"),
-            backgroundColor: theme.fn.themeColor("bg-white"),
+            color: "var(--mb-color-text-primary)",
+            borderColor: "var(--mb-color-border)",
+            backgroundColor: "var(--mb-color-background)",
             "&:hover": {
-              color: primaryColor,
-              backgroundColor: theme.fn.themeColor("bg-light"),
+              color: "var(--mb-color-text-hover)",
```

Comment:
is using var somehow better than theme? I thought with theme we have some TS support

Reply:
I'm not up to date with all the colors thing yet, but if no new colors are added to frontend/src/metabase/ui/utils/colors.ts, then it won't be available in Mantine either via `theme.fn.themeColor` or via props e.g. (`c`, `bg`, etc).

Most recent added colors are in the form of CSS custom properties and they're overridden in `EmbedFrame.module.css` like this:
https://github.com/metabase/metabase/blob/bda5fb7ae24a0bac8138405253dd7f25aecad37e/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.module.css#L53-L62

these colors also aren't available in Mantine.